context/app/static/js/components/Routes/Routes.jsx

```diff
@@ -65,7 +64,7 @@ function Routes({ flaskData }) {
   if (urlPath.startsWith('/browse/donor/')) {
     return (
       <Route>
-        <Donor assayMetadata={entity} vitData={vitessce_conf} />
```

Comment:
The `vitData` prop was removed, but it's not reintroduced elsewhere by context. Was this an unused prop?

Reply:
Yes, that is correct! I removed/am removing props that are currently not being used if that is okay.